metadata-ingestion/src/datahub/ingestion/source/kafka_connect/kafka_connect.py

```diff
@@ -121,7 +121,9 @@ def get_connectors_manifest(self) -> Iterable[ConnectorManifest]:
                     connector_manifest.config, self.config.provided_configs
                 )
             connector_manifest.url = connector_url
-            connector_manifest.topic_names = self._get_connector_topics(connector_name)
+            connector_manifest.topic_names = self._get_connector_topics(
```

Comment:
nit - ideally when calling this method, we'd use kwargs e.g. `self._get_connector_topics(connector=..., type=...)`

Reply:
Ideally, this should become a lint checker
Unfortunately, this is not supported by `ruff` yet https://github.com/astral-sh/ruff/issues/3269